src/components/Articles.jsx

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+import { ArticlesCard } from "./ArticlesCard";
+import { useEffect, useState } from "react";
+import { Row } from "react-bootstrap";
+
+export function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get("https://sakhee-news.onrender.com/api/articles").then((res) => {
```

Comment:
I would look at extracting your api calls into functions that you import into the component - it will really tidy up your code 👍